app/models/user.rb

```diff
@@ -1,8 +1,11 @@
+# frozen_string_literal: true
+
 class User < ApplicationRecord
   include User::Relationship
   include User::Relationship::Follow
   include User::Relationship::Block
   include User::AnswerMethods
+  include User::BanMethods
```

Comment:
hmm.. seeing these `has_many :whatevers` down there makes me think if it would make sense to move them to the `User::WhateverMethods` module (might need to wrap them inside a `included do .. end` block), and renaming these modules to `User::Whatevers`.  But this is nothing for this PR ;-)